src/pages/LEVANTE/RegisterUsers.vue

```diff
@@ -155,6 +155,34 @@ const allFields = [
     dataType: 'string',
   },
 ];
+// Used to remove all other fields in the uploaded CSV
+const allFieldsIdsList = allFields.map((a) => a.field);
+const sanitizeAndValidateSubmittedUsers = (users) => {
+  const userIdMap = {};
+  const filteredFieldsUsers = users.map((user) => {
+    const acc = {};
+    Object.keys(user).forEach((uKey) => {
+      if (allFieldsIdsList.includes(uKey)) {
+        acc[uKey] = user[uKey];
+      }
+    });
+    // Allowing 0 as a valid id
+    const idList = [user['childId'], user['teacherId'], user['parentId']].filter((a) => 0 || !!a);
+    if (idList.length > 1) {
+      addErrorUser(user, `Multiple ids present for the user: ${user.id}`);
+      throw new Error(`Mutliple ids found for user, please see the table below for more details`);
```

Comment:
This error message is a little confusing. For the bad row input, it's hard to decipher what actually needs to be changed. There are 3 things wrong with this input. 1. The parent id does not exist. 2. It is a Teacher so they should not have the teacher id column filled out. 3. They have month and birth year which a Teacher should not have. There is a lot of validation needed here but we should split it into multiple PRs. For now, let's just change the message to something like "One or more columns contain incorrect values. Please review and correct the data."

Input:
![Screenshot 2024-06-06 at 12 30 02 PM](https://github.com/yeatmanlab/roar-dashboard/assets/80180474/b85cb965-8dad-40a4-bd10-84c7f41d41cd)

Output: 
![Screenshot 2024-06-06 at 12 30 13 PM](https://github.com/yeatmanlab/roar-dashboard/assets/80180474/fa59752b-d384-4f82-be59-afa85de7255f)